src/components/react-json-schemaForm/GeometryField/styles.scss

```diff
@@ -70,40 +70,102 @@
 }
 
 .mapbox-gl-path-container {
-  display: flex;
-  align-items: center;
+  float: left;
   pointer-events: auto;
   user-select: none;
   background-color: #ffffff;
   color: #182026;
-  padding: 4px 6px;
+  padding: 4px 8px;
   margin: 10px;
   border-radius: 4px;
   box-shadow: 0 0 0 2px rgba(0, 0, 0, 0.1);
+}
 
-  input {
-    margin-right: 4px;
-  }
+.mapbox-gl-path-theme-selection__checkbox-label {
+  cursor: pointer;
+  padding-left: 5px;
+}
 
-  select {
-    margin-left: 8px;
-  }
+.mapbox-gl-path-theme-selection {
+  height: 0;
+  opacity: 0;
+  overflow: hidden;
+  transition: all 0.3ms ease;
 }
 
-.mapbox-gl-path-popup {
-  button {
-    appearance: none;
-  }
+.mapboxgl-popup-content {
+  padding: 1.2rem 0.8rem 0.8rem !important;
```

Comment:
Do you really need this "!important"?

Reply:
Everything i typed in the basecode is important, but ok i check that :-)

Reply:
Indeed, i just removed it 👍